shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -166,6 +175,28 @@ private void popSystemNavigator() {
         mActivity.finish();
     }
 
+    private JSONObject getClipboardData(String format) throws JSONException {
+        ClipboardManager clipboard = (ClipboardManager) mActivity.getSystemService(Context.CLIPBOARD_SERVICE);
+        ClipData clip = clipboard.getPrimaryClip();
+        if (clip == null)
+            return null;
+
+        if ((format == null || format.equals(kTextPlainFormat)) &&
+            clip.getDescription().hasMimeType(ClipDescription.MIMETYPE_TEXT_PLAIN)) {
+              JSONObject result = new JSONObject();
+              result.put("text", clip.getItemAt(0).getText().toString());
```

Comment:
The mime type check only verifies that the clip data contains plain text. Are you certain the item at index 0 contains that plain text information? Must admit, not very familiar with the API.